core/domain/auth_services_test.py

```diff
@@ -237,59 +236,43 @@ def test_delete_association_when_it_is_missing_does_not_raise(self) -> None:
         # Should not raise.
         auth_services.delete_external_auth_associations('does_not_exist')
 
-    def test_auth_session_established_or_destoryed(self) -> None:
-        auth_section = []
-        def mock_establish_auth_session(
-            _: webapp2.Request,
-            __: webapp2.Response
-        ) -> None:
-            auth_section.append('established')
-
-        def mock_destroy_auth_session(
-            _: webapp2.Response
-        ) -> None:
-            auth_section.remove('established')
-
-        with self.swap(
-            platform_auth_services,
-            'establish_auth_session',
-            mock_establish_auth_session
-        ):
-            auth_services.establish_auth_session(
-                webapp2.Request.blank('/'),
-                webapp2.Response()
+    @mock.patch(
+            'core.domain.auth_services.platform_auth_services.'
```

Comment:
Deindent this and the next line by 4. Deindent the line below it by 8 to match the opening paren.